packages/enhanced/src/lib/sharing/ConsumeSharedModule.ts

```diff
@@ -78,6 +78,14 @@ export type ConsumeOptions = {
    * include the fallback module in a sync way
    */
   eager: boolean;
+  /**
+   * Share a specific layer of the module, if the module supports layers.
+   */
+  layer?: string | null;
```

Comment:
i need to see about layer/issuerlayer 

we can add layer, but this will not make match unless it is same as consumed module - however i can get issuerLayer and build request map with that?